libs/serviceproxy.py

```diff
@@ -131,8 +132,27 @@ def getEvent(self):
                 event = jsonhelpers.decode(servicedefs.DiagnosticEvent, evJsonStr)
         return event
 
-
     def saveto(self, path, alternatePath):
         req = servicedefs.SavetoRequest(self.incrSeq(), servicedefs.ReloadRequestArgs(path, alternatePath))
         jsonStr = jsonhelpers.encode(req)
         self.__comm.postCmd(jsonStr)        
+
+    def navTo(self, searchText, fileName):
+        req = self.wrap_req_dict ({
+            'command': 'navto',
+            'arguments': {
+                'searchValue': searchText,
+                'file': fileName,
+                'maxResultCount': 20
+            }
+        })
+        resp_dict = self.__comm.sendCmdSync(json.dumps(req), req['seq'])
```

Comment:
Let's test this on a large project.  If this is sync, then this is blocking.  We need to make sure this isn't too sluggish.